src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,6 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public abstract boolean isMutating();
```

Comment:
Missing header comment. All non-trivial classes and methods should have javadoc format header comments.